src/AbpHelper.Core/Templates/Crud/Groups/Test/test/{{ProjectInfo.FullName}}.Domain.Tests/{{EntityInfo.RelativeDirectory}}/{{EntityInfo.Name}}DomainTests.cs

```diff
@@ -2,24 +2,23 @@
 using Shouldly;
 using Xunit;
 
-namespace {{ EntityInfo.Namespace }}
+namespace {{ EntityInfo.Namespace }};
+public class {{ EntityInfo.Name }}DomainTests : {{ ProjectInfo.Name }}DomainTestBase
 {
-    public class {{ EntityInfo.Name }}DomainTests : {{ ProjectInfo.Name }}DomainTestBase
+    public {{ EntityInfo.Name }}DomainTests()
     {
-        public {{ EntityInfo.Name }}DomainTests()
-        {
-        }
+    }
 
-        /*
-        [Fact]
-        public async Task Test1()
-        {
-            // Arrange
+    /*
+    [Fact]
+    public async Task Test1()
+    {
+        // Arrange
 
-            // Assert
+        // Assert
 
-            // Assert
-        }
-        */
+        // Assert
     }
+    */
+}
 }
```

Comment:
Should be remove `}`.

![image](https://user-images.githubusercontent.com/13699353/169793543-a9342829-e6e6-4617-a8f8-3937f6d0e5f0.png)